api/envoy/config/core/v3/health_check.proto

```diff
@@ -126,6 +126,14 @@ message HealthCheck {
     // <envoy_v3_api_msg_type.matcher.v3.StringMatcher>`. See the :ref:`architecture overview
     // <arch_overview_health_checking_identity>` for more information.
     type.matcher.v3.StringMatcher service_name_matcher = 11;
+
+    // Specifies a list of HTTP response statuses considered retriable. If provided, responses in this range
+    // will count towards the configured :ref:`unhealthy_threshold <envoy_v3_api_field_config.core.v3.HealthCheck.unhealthy_threshold>`.
+    // By default all responses not in :ref:`expected_statuses <envoy_v3_api_field_config.core.v3.HealthCheck.HttpHealthCheck.expected_statuses>`
+    // will result in the host being considered immediately unhealthy. Ranges follow half-open semantics of
```

Comment:
IIUC the main difference is what's counted as immediately unhealthy. Can you please update the comment to describe what is immediately unhealthy.

Reply:
Sorry, could you add a bit more on what's unclear here? It mentions these ranges counting towards unhealthy threshold and they says everything everything not in expected statuses by default is considered immediately unhealthy.

Reply:
I think the following emphasizes that the field is related to hosts not considered immediately unhealthy (the name retriable implies it, but I think this is more explicit):
```suggestion
    // Specifies a list of HTTP response statuses considered retriable. If provided, responses in this range
    // will count towards the configured :ref:`unhealthy_threshold <envoy_v3_api_field_config.core.v3.HealthCheck.unhealthy_threshold>`, and will not result in the host being considered immediately unhealthy
    // (By default all responses not in :ref:`expected_statuses <envoy_v3_api_field_config.core.v3.HealthCheck.HttpHealthCheck.expected_statuses>`
    // will result in the host being considered immediately unhealthy). Ranges follow half-open semantics of
```